modules/standard/Sort.chpl

```diff
@@ -24,45 +24,103 @@
 // TODO -- performance test sort routines and optimize (see other TODO's)
 /*
 
-Supports standard algorithms for sorting data.
+The sort module provides functions to sort arrays and lists. It is designed to
```

Comment:
small nit, make sure this line renders right in the module index

Reply:
Could you clarify what you mean by module index? is it just the modules/standard.html page? This line doesn't render there.

Reply:
I mean this page, https://chapel-lang.org/docs/main/chpl-modindex.html